services/infinispan-device-registry/src/main/java/org/eclipse/hono/registry/infinispan/InfinispanRegistryConfig.java

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.registry.infinispan;
+
+import org.eclipse.hono.deviceregistry.ApplicationConfig;
```

Comment:
IMHO we should not have a dependency on another registry implementation.

Reply:
deviceregistry package contains a number of useful classes, not related only to file-based registry. We can move those to service-base module? Or move file-based registry into its own package (and the dummy one)?
As for the ApplicationConfig we can refactor it to remove all file-based registry specific stuff from it.